project/src/views/evaluations/evaluations.js

```diff
@@ -49,10 +49,8 @@ const EvaluationsTable = function(options = {}) {
 }
 
 const EvaluationsTableHeader = function(options = {}) {    
-    const headings = [];
-    options.headings.forEach(function(el) {
-        const generateEl = `<th class="evaluations-tbl-head evaluations-tbl-head-skin">${el}</th>`;
-        headings.push(generateEl);
+    const headings = options.headings.map(function(el) {
+        return `<th class="evaluations-tbl-head evaluations-tbl-head-skin">${el}</th>`;
     })
     const headingsEl = `${headings.join('')}`;
```

Comment:
Why the extra wrapping in a template string? Also, not a big deal, but `headings` is actually a superfluous variable. You could have simply said:
```javascript
const headingsEl = options.headings.map(function(el) {
  return `<th class="evaluations-tbl-head evaluations-tbl-head-skin">${el}</th>`;
}).join('');
```

Reply:
I didn't see this one, generally I tried to replace the forEach with map. I can also change this function into an arrow function.